urllib3/connection.py

```diff
@@ -56,7 +56,12 @@ class ConnectionError(Exception):
     'https': 443,
 }
 
-RECENT_DATE = datetime.date(2014, 1, 1)
+# When updating RECENT_DATE be sure to not
+# update the value to be later than two years
+# in the past and no earlier than 6 months
+# in the past to stay consistent with the
+# history of this value.
```

Comment:
Thanks, yeah that was my third revision of that comment before I just decided to let it be reviewed and hopefully come up with a better way of putting it. :) I'll change this!